gallery/src/styles/NxList/NxListErrorStateExample.tsx

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxLoadError } from '@sonatype/react-shared-components';
+
+const NxListErrorExample = () =>
+  <>
+    <h3 className="nx-h3">
+      List with error alert
+    </h3>
+    <ul className="nx-list">
+      <li className="nx-list__item nx-list__item--error">
+        <NxLoadError error=""/>
```

Comment:
I'd add a retry button to this example as it would be expected in real usage.  See the `NxLoadError` docs for a description of when it would be acceptable not to have a retry button (rarely).  The button is added when the `retryHandler` prop is specified.

Reply:
> f369f03

That example now has some flaws in that it should really show an empty row if the error is cleared but there's still no data.  However, there's a simpler solution that I'd find acceptable: if the retryHandler was simply a no-op function (`() => {}`) so that the button is there in the example but the error doesn't actually go away.

Reply:
> > [f369f03](https://github.com/sonatype/sonatype-react-shared-components/commit/f369f03dc72c8c97447e54d4e887a5afad4efda9)
> 
> That example now has some flaws in that it should really show an empty row if the error is cleared but there's still no data. However, there's a simpler solution that I'd find acceptable: if the retryHandler was simply a no-op function (`() => {}`) so that the button is there in the example but the error doesn't actually go away.

https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/fc2bf3f24ba85ab3c93d46417289b674ecccf059